TAO/orbsvcs/orbsvcs/PortableGroup/PG_Object_Group.cpp

```diff
@@ -880,11 +880,11 @@ TAO::PG_Object_Group::minimum_populate (void)
 {
   ACE_GUARD (TAO_SYNCH_MUTEX, guard, this->internals_);
 
-  if ( this->get_membership_style () == PortableGroup::MEMB_INF_CTRL)
+  if ( this->get_membership_style () == PortableGroup::MEMB_INF_CTRL )
     {
       PortableGroup::MinimumNumberMembersValue minimum_number_members =
         this->get_minimum_number_members ();
-      if (((PortableGroup::InitialNumberMembersValue)members_.current_size ()) < minimum_number_members)
+      if (members_.current_size () < minimum_number_members)
```

Comment:
these casts are for conversion warnings, in 64bit builds one is a 64bit type, other 32bit